drivers/include/drivers/SerialBase.h

```diff
@@ -119,6 +119,12 @@ class SerialBase : private NonCopyable<SerialBase> {
      */
     void send_break();
 
+    /** Flush hardware buffers associated with serial port
+     *
+     *  @return         0 on success, negative error code on failure
+     */
+    int sync();
```

Comment:
Why not call this function `flush` to better explain what it does?
A `sync` function could have different meaning, typically ensuring that buffered data is written to the physical device.

Reply:
in this case the description is misleading, the goal of this function is to ensure that all data stored in the tx FIFO or TX register is written to the serial bus. I'll update the description to avoid misunderstandings. Thanks!

Reply:
The misleading description was inherited from `FileHandle.h`, which uses the same wording. Could you fix that at the same time?

Reply:
I've updatede the functions description here 6f6b6653a4bf34d923460779566dc206d69bd4ee and here bf8bf698e3381ee0f1acc2aa1d1dab7dc5eae2f2. i hope i've got the correct one.